### PR TITLE
Update installation docs to latest version of composer-asset-plugin

### PR DESCRIPTION
**Pull request by usecomposable**
[ci skip]

Anyone stuck on 1.0.0 will be riddled with deprecation notices.

### CODE REVIEW

File: docs/guide/start-installation.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ by running `composer self-update`.
 
 With Composer installed, you can install Yii by running the following commands under a Web-accessible folder:
 
-    composer global require "fxp/composer-asset-plugin:~1.0.0"
+    composer global require "fxp/composer-asset-plugin:~1.0.3"
```
Comment (klimov-paul) [inline docs/guide/start-installation.md]: Shouldn't it be `~1.0` then?

Comment (df2) [inline docs/guide/start-installation.md]: Depends on what they do with 1.1 ;-) But yeah, that also would pick up 1.0.3 as the latest.

Comment (samdark) [inline docs/guide/start-installation.md]: `~1.0.0` works fine i.e. it will install 1.0.3 and 1.0.4 when it will be released.

**Comment by klimov-paul** [comment]
Adjusted b107d61aac1684e940749abaa1a8aa1241734c76

**Comment by samdark** [comment]
:+1:

**Pull request merged**